package.json

```diff
@@ -5,58 +5,63 @@
   "private": true,
   "author": "BigCommerce",
   "license": "MIT",
+  "dependencies": {
+    "@bigcommerce/stencil-utils": "^1.0.7",
+    "async": "^2.5.0",
+    "babel-polyfill": "^6.26.0",
+    "fastclick": "^1.0.6",
+    "foundation-sites": "^5.5.3",
+    "grunt": "^0.4.5",
+    "html5-history-api": "^4.2.7",
+    "jquery": "^2.2.4",
+    "jquery-zoom": "^1.7.15",
+    "jstree": "vakata/jstree",
+    "lazysizes": "3.0.0",
+    "lodash": "^4.17.4",
+    "nod-validate": "^2.0.12",
+    "pace": "hubspot/pace#a03f1f1de62c9cea6c88b2267b8d7a83858b6cb6",
```

Comment:
Do we know why pace is locked down to a commit & not to an actual release ?

Reply:
@junedkazi IIRC it's because they made a breaking change.  i'll do a quick check to see if we can update now

Reply:
yup, there's a breaking change.  let's just create a separate PR with the fix